live-build/hooks/30-fix-timedatectl.chroot

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set -e
+
+mv /usr/bin/timedatectl /usr/bin/timedatectl.real
+
+cat >/usr/bin/timedatectl<<EOF
+#!/bin/sh
+
+# wrapper to make sure timedatectl works with the
+# writable-paths setup on ubuntu core (making the symlink
+# absolute instead of using a relative path)
+# LP: #1650688 for details
+
+set -e
+
+TIMEDATECTL=/usr/bin/timedatectl.real
+
+case $1 in
```

Comment:
i merged #16 anything else for spellchacking needed ? (do you want to run spellintian too ?)
nothe that any spellcheck will chocke on the mentioning of "writable-paths" because the original implementation of it is already missing an "e"